yarn.lock

```diff
@@ -7665,10 +7689,10 @@ ovh-module-emailpro@^7.2.1:
     punycode "^1.2.4"
     validator "^4.0.0"
 
-ovh-module-exchange@^9.3.4:
-  version "9.3.4"
-  resolved "https://registry.yarnpkg.com/ovh-module-exchange/-/ovh-module-exchange-9.3.4.tgz#74901fb074e908ef4f33816ba14c719837ec3a0d"
-  integrity sha512-FtMjeYedSYqLU14YjhI9/N6LIhdAQn70Qt22UtI0nIUsMvx7FaOaT+EWsbKE9yS0Pz+Ss5SsP6h/ACdXFvTJCQ==
+ovh-module-exchange@^9.4.0:
```

Comment:
Is the bump needed for this pull request?

Because it has already be bumped here: #971

Reply:
This was automatically generated when I upgraded `ovh-api-services` to its latest version (rebase on `develop` was made beforehand). 
This is probably due to https://github.com/ovh-ux/ovh-manager-web/pull/971 only commiting `package.json` ?

Reply:
Yes you're right! Thanks!